maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/SurefireHelper.java

```diff
@@ -265,15 +265,18 @@ private static void throwException( SurefireReportParameters reportParameters, R
                                            Exception firstForkException )
             throws MojoFailureException, MojoExecutionException
     {
+        String msg = createErrorMessage( reportParameters, result, firstForkException );
         if ( isFatal( firstForkException ) || result.isInternalError()  )
         {
-            throw new MojoExecutionException( createErrorMessage( reportParameters, result, firstForkException ),
-                                                    firstForkException );
+            throw firstForkException == null
+                ? new MojoExecutionException( msg )
+                : new MojoExecutionException( msg, firstForkException );
```

Comment:
What is the difference 
`MojoExecutionException( msg )` and `MojoExecutionException( msg, null )`
for both `getCause()` will return null